src/FsAutoComplete.Core/Debug.fs

```diff
@@ -102,17 +166,18 @@ module Debug =
               >> logFunctionName eventArgs.Payload.[1]
               >> Log.addContextDestructured "seconds" delta.TotalSeconds
               >> Log.addContextDestructured "message" (eventArgs.Payload.[0])
+              |> Some
             | false, _ ->
               Log.setMessage "Finished {function}: {message}"
               >> logFunctionName eventArgs.Payload.[1]
               >> Log.addContextDestructured "message" eventArgs.Payload.[0]
-          | other ->
-            Log.setMessage "Unknown event {name}({id}) with payload {payload}"
-            >> Log.addContext "id" other
-            >> Log.addContextDestructured "name" eventArgs.EventName
-            >> Log.addContextDestructured "payload" (eventArgs.Payload |> Seq.toList)
-
-        (eventLevelToLogLevel eventArgs.Level) message
+              |> Some
+          | other -> None
```

Comment:
were there messages that were blowing up?

Reply:
Oh yeah adding `dotnet counters` make this extremely verbose.  Can revert and make a better PR later around filtering that noise out.